free-1.md

```diff
@@ -10,41 +10,148 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The language targets musicians interested in digital music synthesis.
+Currently, there are two categories of tools which allow producers to generate
+music using a computer.
+
+The first is a type of software called a Digital Audio Workstation (DAW).
+Popular DAWs like Pro Tools and FL Studio usually have built-in digital
+"instruments" which user's can program by writing a MIDI score. These are
+usually prefabricated instruments that can only be customized by changing
+certain parameters. This makes it possible to get a range of sounds out of each
+instrument, but it is hard or impossible to create completely new instruments.
+
+Musicians interested in creating their own instruments can use a DSL called
+CSound (which is taught at Mudd in MUS 80). CSound allows users to create
+chains of oscillators -- primitive components used to generate a sine wave --
+and effects like filters to literally build from scratch their own digital
+instruments. This simulates the way the earliest synthesizers were used; the
+musician would connect oscillators and other components using wires. CSound
+takes this process and makes it digital.
+
+But there's a problem with this current state of things: DAWs operate at a high
+level, where the primitive unit is an instrument. CSound operates at a much
+lower level, where the primitive unit is an oscillator (this fact and its
+opcode-based syntax means it is used almost exclusively by academics). There is
+a lack of tools in the middle. Musicians who want to create their own
+instruments but don't want to deal with the low level of abstraction offered by
+CSound have little recourse.
```

Comment:
The polarization is really interesting. I'm curious why this kind of difference in abstraction levels exist.